packages/elements/src/chart/index.ts

```diff
@@ -205,6 +205,11 @@ export class Chart extends BasicElement {
    * @returns {ChartOptions} chart config with theme
    */
   protected get themableChartOption (): ChartOptions {
+    let boxHeight = Number(getComputedStyle(this).getPropertyValue('font-size').replace('px', ''));
+    if (this.config?.options?.plugins?.legend?.labels?.usePointStyle) {
+      boxHeight = this.cssVarAsNumber('--legend-key-box-width', '10') as number;
```

Comment:
`--legend-key-box-width` value should be extracted only once. Let's refactor line 240 as well.

Reply:
Updated by creating variable boxWidth.